### PR TITLE
BSC light client contract

### PR DESCRIPTION
**Pull request by furoxr**

### CODE REVIEW

File: .maintain/config/bridge-pangoro-chapel.toml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [pangoro]
-endpoint = "wss://pangoro-rpc.darwinia.network:443"
-relayer_private_key = "//Alice"
-# relayer_real_account = "0x..."
+endpoint = "https://pangoro-rpc.darwinia.network:443"
+bsc_abi_path = "/Users/furoxr/Projects/darwinia-messages-sol/contracts/bridge/abi/src/truth/bsc/BSCLightClient.sol/BSCLightClient.json"
```
Comment (fewensa) [inline .maintain/config/bridge-pangoro-chapel.toml:3]: Bundle abi to code is better.

File: .maintain/config/bridge-pangoro-chapel.toml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 [pangoro]
-endpoint = "wss://pangoro-rpc.darwinia.network:443"
-relayer_private_key = "//Alice"
-# relayer_real_account = "0x..."
+endpoint = "https://pangoro-rpc.darwinia.network:443"
+bsc_address = "0x6ac5ae3fa61b2cbea625dd24f57bdc3d952333c9"
```
Comment (hujw77) [inline .maintain/config/bridge-pangoro-chapel.toml:3]: Please follow the latest contract address changes.
https://github.com/darwinia-network/darwinia-messages-sol/blob/4a170944b844567a6cc897b264c0ade5084a3388/contracts/bridge/bin/addr/test/pangoro.json#L14

Comment (furoxr) [reply .maintain/config/bridge-pangoro-chapel.toml]: Updated, and I've done some tests.

```2022-07-13 02:21:33 TRACE pangoro-chapel: Current checkpoint on Pangoro: 20960200
2022-07-13 02:21:33 TRACE pangoro-chapel: Current finalized authority set length on Pangoro: 7
2022-07-13 02:21:33 TRACE pangoro-chapel: Current block number on Chapel: 21011533
2022-07-13 02:21:34 TRACE pangoro-chapel: Relaying headers[20960400:20960403] to Pangoro
2022-07-13 02:21:35  INFO pangoro-chapel: Sending tx: 0xb8b4d89acc1cc719ec6e6d57f8cf6fa5c3d9739c88ba6b4f4f3e02399042f26c
2022-07-13 02:21:42 TRACE pangoro-chapel: Current checkpoint on Pangoro: 20960400
2022-07-13 02:21:42 TRACE pangoro-chapel: Current finalized authority set length on Pangoro: 7
2022-07-13 02:21:42 TRACE pangoro-chapel: Current block number on Chapel: 21011536
2022-07-13 02:21:43 TRACE pangoro-chapel: Relaying headers[20960600:20960603] to Pangoro
2022-07-13 02:21:44  INFO pangoro-chapel: Sending tx: 0x9fde5ba17e17dde39c9409df2b969f05ee4af8007a6edd9e89a86c4a3fc0d711
2022-07-13 02:21:51 TRACE pangoro-chapel: Current checkpoint on Pangoro: 20960600
2022-07-13 02:21:51 TRACE pangoro-chapel: Current finalized authority set length on Pangoro: 7
2022-07-13 02:21:51 TRACE pangoro-chapel: Current block number on Chapel: 21011539
2022-07-13 02:21:51 TRACE pangoro-chapel: Relaying headers[20960800:20960803] to Pangoro
2022-07-13 02:21:52  INFO pangoro-chapel: Sending tx: 0xc9db2d6e5360b0efdbd5823484c8f519a04b9847d97e055b7f04734e81a8e335
2022-07-13 02:21:59 TRACE pangoro-chapel: Current checkpoint on Pangoro: 20960800
2022-07-13 02:21:59 TRACE pangoro-chapel: Current finalized authority set length on Pangoro: 7
2022-07-13 02:22:00 TRACE pangoro-chapel: Current block number on Chapel: 21011542
2022-07-13 02:22:00 TRACE pangoro-chapel: Relaying headers[20961000:20961003] to Pangoro
2022-07-13 02:22:01  INFO pangoro-chapel: Sending tx: 0x66059958cb6d4424e34759db69698206bd7facd8b478945ac6c34a58909b5e76
2022-07-13 02:22:08 TRACE pangoro-chapel: Current checkpoint on Pangoro: 20961000
2022-07-13 02:22:08 TRACE pangoro-chapel: Current finalized authority set length on Pangoro: 7
2022-07-13 02:22:08 TRACE pangoro-chapel: Current block number on Chapel: 21011545
2022-07-13 02:22:08 TRACE pangoro-chapel: Relaying headers[20961200:20961203] to Pangoro
2022-07-13 02:22:09  INFO pangoro-chapel: Sending tx: 0x874866595fff289ff772e8ce533bf2446f475e411fb0ccac30d6c4a1a086d6d6
2022-07-13 02:22:17 TRACE pangoro-chapel: Current checkpoint on Pangoro: 20961200
2022-07-13 02:22:17 TRACE pangoro-chapel: Current finalized authority set length on Pangoro: 7
2022-07-13 02:22:17 TRACE pangoro-chapel: Current block number on Chapel: 21011548
2022-07-13 02:22:17 TRACE pangoro-chapel: Relaying headers[20961400:20961403] to Pangoro
2022-07-13 02:22:19  INFO pangoro-chapel: Sending tx: 0x32357c1ad883cd5d752e444f7e4c328bb59718a45ba9b55da21a67b11bb16890```

**Review by hujw77 — COMMENTED** [review]
LGTM

**Pull request merged**